test/system/helpers.bash

```diff
@@ -747,7 +747,7 @@ function parse_table() {
 function random_string() {
     local length=${1:-10}
 
-    head /dev/urandom | tr -dc a-zA-Z0-9 | head -c$length
+    tr -dc a-zA-Z0-9 < /dev/urandom | head -c$length
```

Comment:
@edsantiago PTAL

Reply:
THB I don't know what that head does there. Why it was there?

Head takes 10 lines from a file, that makes as sense for text files not /dev/urandom.
For all we know first 10 chars in /dev/urandom may be new lines (I know, it is really unlikely) meaning `random_string` would return empty string.

`random_string 1000` is returning random length string not strings 1000 chars long.

Reply:
old verison:
```
for i in $(seq 10); do random_string 1000 | wc -c; done
686
468
512
735
288
954
749
458
463
891
```

Reply:
The first `head` was there because it's Good Enough™.